streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBGenericOptionsToDbOptionsColumnFamilyOptionsAdapterTest.java

```diff
@@ -119,7 +119,7 @@ private void verifyDBOptionsMethodCall(final Method method) throws Exception {
         } catch (final InvocationTargetException undeclaredMockMethodCall) {
             assertThat(undeclaredMockMethodCall.getCause(), instanceOf(AssertionError.class));
             assertThat(undeclaredMockMethodCall.getCause().getMessage().trim(),
-                matchesPattern("Unexpected method call DBOptions\\." + method.getName() + "((.*\n*)*):"));
+                matchesPattern("Unexpected method call DBOptions\\." + method.getName() + "((.*\n*)*):.*"));
```

Comment:
Maybe it's worth to simplify to `startsWith("Unexpected method call DBOptions." + method.getName())` ?

Reply:
That works for me, I'll update the PR accordingly